src/Service/Sqs/src/Input/CreateQueueRequest.php

```diff
@@ -294,22 +298,26 @@ private function requestBody(): array
         }
         $payload['QueueName'] = $v;
         if (null !== $v = $this->attributes) {
-            $index = 0;
-            foreach ($v as $mapKey => $mapValue) {
-                if (!QueueAttributeName::exists($mapKey)) {
-                    throw new InvalidArgument(sprintf('Invalid key for "%s". The value "%s" is not a valid "QueueAttributeName".', __CLASS__, $mapKey));
+            if (empty($v)) {
+                $payload['Attributes'] = new \stdClass();
+            } else {
+                $payload['Attributes'] = [];
+                foreach ($v as $name => $mv) {
+                    if (!QueueAttributeName::exists($mapKey)) {
```

Comment:
`$name` not `$mapKey`.

Reply:
the bot will not automatically fix bugs in the code generator by reading review comments...